lib/dnsimple/client/webhooks.rb

```diff
@@ -0,0 +1,103 @@
+module Dnsimple
+  class Client
+    module Webhooks
+
+      # Lists the webhooks in the account.
+      #
+      # @see https://developer.dnsimple.com/v2/webhooks/#list
+      # @see #all_webhooks
+      #
+      # @example List all webhooks
+      #   client.webhooks.list(1010)
+      #
+      # @param  [Fixnum] account_id the account ID
+      # @param  [Hash] options the filtering and sorting option
+      # @return [Dnsimple::CollectionResponse<Dnsimple::Struct::Webhook>]
+      #
+      # @raise  [Dnsimple::RequestError]
+      def webhooks(account_id, options = {})
+        response = client.get(Client.versioned("/%s/webhooks" % [account_id]), options)
+
+        Dnsimple::CollectionResponse.new(response, response["data"].map { |r| Struct::Webhook.new(r) })
+      end
+      alias :list :webhooks
+      alias :list_webhooks :webhooks
+
+      # Lists ALL the webhooks in the account.
+      #
+      # This is equal to {#webhooks}.
+      # We have implemented both methods to have a consistent design with other
+      # resources that implements pagination.
+      #
+      # @see https://developer.dnsimple.com/v2/webhooks/#list
+      # @see #webhooks
+      #
+      # @param  [Fixnum] account_id the account ID
+      # @param  [Hash] options the filtering and sorting option
+      # @return [Dnsimple::CollectionResponse<Dnsimple::Struct::Webhook>]
+      #
+      # @raise  [Dnsimple::RequestError]
+      def all_webhooks(account_id, options = {})
```

Comment:
@weppos I added it for consistency and explained it in the code comment: https://github.com/aetrion/dnsimple-ruby/pull/82#discussion-diff-54880203R29

I consider it as an alias for now. Do we need to remove it anyway?